src/tree/rules/RulesTreeItem.ts

```diff
@@ -37,7 +37,11 @@ export class RulesTreeItem extends AzExtParentTreeItem {
         this.rulesFolderPath = this.getRulesFolderPath();
 
         if (!await hasFiles(this.rulesFolderPath)) {
-            await this.exportRulesToLocalFolder(this.rulesFolderPath);
+            const base64ZipContent = await this.exportRuleset();
+            if (!base64ZipContent) {
+                return [];
```

Comment:
do we need to record this telemetry?

Reply:
The `getchildren` event will log the children count.